src/test/java/com/michelin/ns4kafka/integration/TopicIntegrationTest.java

```diff
@@ -601,6 +599,13 @@ void shouldDeleteRecordsOnCompactTopic() {
         assertEquals(HttpStatus.UNPROCESSABLE_ENTITY, exception.getStatus());
     }
 
+    private void forceTopicSynchronization() throws InterruptedException {
+        topicAsyncExecutorList.forEach(TopicAsyncExecutor::run);
+
+        // Wait for topics to be updated in Kafka broker
+        Thread.sleep(2000);
```

Comment:
This sleep patches KRaft-related unstability or general unstability?

Reply:
Well it happens after migrating, in `TopicIntegrationTest#shouldUpdateTopic`, where it takes some time for the broker to update the topic configuration despite Ns4Kafka topic executor ended the synchronization successfully.

It precisely happens when checking the configuration update here:
- https://github.com/michelin/ns4kafka/blob/948322f04531d12adff4b289221a18d3e6e52ba2/src/test/java/com/michelin/ns4kafka/integration/TopicIntegrationTest.java#L342
- https://github.com/michelin/ns4kafka/blob/948322f04531d12adff4b289221a18d3e6e52ba2/src/test/java/com/michelin/ns4kafka/integration/TopicIntegrationTest.java#L385

Without sleeping, `retention.ms` can be still equal to `60000` (like 1 time over 10...)

I'm still wondering how it can be related to KRaft

Reply:
Ok for me then